src/k8s-extension/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 1.3.0
 ++++++++++++++++++
 * Add support for provisionedClusters
+* microsoft.azureml.kubernetes: Always show TSG link for AzureMLKubernetes extension at the head.
```

Comment:
This change would be part of the version release 1.3.1

Reply:
changed